scripts/hostapd/install

```diff
@@ -102,6 +102,17 @@ if [ -f /etc/dhcpcd.conf ]; then
     sudo echo "denyinterfaces wlan0" | sudo tee --append /etc/dhcpcd.conf > /dev/null
 fi
 
+
+# Pin internal wireless adapter to wlan0 on boot
+interfaces=$(ls /sys/class/net/ | grep wlan)
+for interface in $interfaces; do
+    interfaceDriver="$(basename $(readlink /sys/class/net/$interface/device/driver))";
+    if [ "$interfaceDriver" == 'xradio_wlan' ] || [ "$interfaceDriver" == 'brcmfmac' ]; then
+        mac=$(cat /sys/class/net/$interface/address | tr -d '\n')
+	echo "SUBSYSTEM==\"net\", ACTION==\"add\", ATTRS{address}==\"$mac\", KERNEL==\"wlan*\", NAME=\"wlan0\"" |  sudo tee  /etc/udev/rules.d/40-onboard-wlan-pin.rules 
```

Comment:
Tab and misalignment

Reply:
What's with the extra spaces around `sudo tee` before and after?

Reply:
The `sudo tee` is wrong this doesn't do anything. You need the `--append` and pipe output to /dev/null please see other places.

Reply:
This is a temporary solution right? We may put USB adapters with these drivers and there would be two `wlan0`?